demo/kserve/scripts/install/check-env-variables.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+# Environment variables
+# - CHECK_UWM: Set this to "false", if you want to skip the User Workload Configmap check message
+# - TARGET_OPERATOR: Set this among odh, rhods or brew, if you want to skip the question in the script.
+set -o pipefail
+set -o nounset
+set -o errtrace
+# set -x   #Uncomment this to debug script.
+
+source "$(dirname "$(realpath "$0")")/../env.sh"
+source "$(dirname "$(realpath "$0")")/../utils.sh"
+if [[ -n "${CHECK_UWM+x}" && ${CHECK_UWM} == "false" ]]
+then
+  input="y"
+else
+  echo "** Check User Workload Configmap for Kserve metrics before you execute this script **"
+  echo 
+  cat <<EOF
+
+  $ oc get cm cluster-monitoring-config -n openshift-monitoring -o yaml
+
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: cluster-monitoring-config
+    namespace: openshift-monitoring
+  data:
+    config.yaml: |
+      enableUserWorkload: true    #<==== Check this part
+
+  ---
+
+  $ oc get cm user-workload-monitoring-config -n openshift-user-workload-monitoring -o yaml
+
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: user-workload-monitoring-config
+    namespace: openshift-user-workload-monitoring
+  data:
+    config.yaml: |
+      prometheus:
+        logLevel: debug 
+        retention: 15d #Change as needed  <==Check this
+EOF
+
+  echo
+  read -p "Have you checked if user workload configmap is set correctly? (then enter 'y')" input
+fi
+
+if [ "$input" != "y" ];
+then
+    echo "ERROR: Please check the configmap and execute this script again"
+    exit 1
+fi
+
+if [[ -n "${DEPLOY_ODH_OPERATOR+x}" ]]
+then  
+  deploy_odh_operator=${DEPLOY_ODH_OPERATOR}
+fi
```

Comment:
I think this is missing the code to raed the value of the variable from user input, like done with other variables in this script

Reply:
The default value is true so if the user does not set this variable, it will deploy odh operator. I don't think we need to check the variable every time because this script was designed to install odh operator. 
This is like `CUSTOM_MANIFESTS_URL` variable.

Reply:
okay